### ISSUE TITLE
Bionic is signed using old GPG key

### ISSUE THREAD

**Issue by C0rn3j** [Icinga/icinga-packaging#308]
It seems the key on https://packages.icinga.com/ubuntu/dists/icinga-bionic/ was rotated:

```
Our GPG key has been changed on September 30, 2024:

pub   rsa4096 2024-05-21 [SC]
      DD3A F619 8ED0 00B4 C0B7 3956 CC11 6F55 AA7F 2382
uid           Icinga GmbH (Build server) <info@icinga.com>
```

But not all packages are signed with the new key on https://packages.icinga.com/icinga.key:

```bash
W: GPG error: https://packages.icinga.com/ubuntu icinga-bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C6E319C334410682
```

They are still signed using the old key:
```gpg
-----BEGIN PGP PUBLIC KEY BLOCK-----
Version: GnuPG v2.0.19 (GNU/Linux)

mQGiBFKHzk4RBACSHMIFTtfw4ZsNKAA03Gf5t7ovsKWnS7kcMYleAidypqhOmkGg
0petiYsMPYT+MOepCJFGNzwQwJhZrdLUxxMSWay4Xj0ArgpD9vbvU+gj8Tb02l+x
SqNGP8jXMV5UnK4gZsrYGLUPvx47uNNYRIRJAGOPYTvohhnFJiG402dzlwCg4u5I
1RdFplkp9JM6vNM9VBIAmcED/2jr7UQGsPs8YOiPkskGHLh/zXgO8SvcNAxCLgbp
BjGcF4Iso/A2TAI/2KGJW6kBW/Paf722ltU6s/6mutdXJppgNAz5nfpEt4uZKZyu
oSWf77179B2B/Wl1BsX/Oc3chscAgQb2pD/qPF/VYRJU+hvdQkq1zfi6cVsxyREV
k+IwA/46nXh51CQxE29ayuy1BoIOxezvuXFUXZ8rP6aCh4KaiN9AJoy7pBieCzsq
d7rPEeGIzBjI+yhEu8p92W6KWzL0xduWfYg9I7a2GTk8CaLX2OCLuwnKd7RVDyyZ
yzRjWs0T5U7SRAWspLStYxMdKert9lLyQiRHtLwmlgBPqa0gh7Q+SWNpbmdhIE9w
ZW4gU291cmNlIE1vbml0b3JpbmcgKEJ1aWxkIHNlcnZlcikgPGluZm9AaWNpbmdh
Lm9yZz6IYAQTEQIAIAUCUofOTgIbAwYLCQgHAwIEFQIIAwQWAgMBAh4BAheAAAoJ
EMbjGcM0QQaCgSQAnRjXdbsyqziqhmxfAKffNJYuMPwdAKCS/IRCVyQzApFBtIBQ
1xuoym/4C7kCDQRSh85OEAgAvPwjlURCi8z6+7i60no4n16dNcSzd6AT8Kizpv2r
9BmNBff/GNYGnHyob/DMtmO2esEuVG8w62rO9m1wzzXzjbtmtU7NZ1Tg+C+reU2I
GNVu3SYtEVK/UTJHAhLcgry9yD99610tYPN2Fx33Efse94mXOreBfCvDsmFGSc7j
GVNCWXpMR3jTYyGj1igYd5ztOzG63D8gPyOucTTl+RWN/G9EoGBv6sWqk5eCd1Fs
JlWyQX4BJn3YsCZx3uj1DWL0dAl2zqcn6m1M4oj1ozW47MqM/efKOcV6VvCs9SL8
F/NFvZcH4LKzeupCQ5jEONqcTlVlnLlIqId95Z4DI4AV9wADBQf/S6sKA4oH49tD
Yb5xAfUyEp5ben05TzUJbXs0Z7hfRQzy9+vQbWGamWLgg3QRUVPx1e4IT+W5vEm5
dggNTMEwlLMI7izCPDcD32B5oxNVxlfj428KGllYWCFj+edY+xKTvw/PHnn+drKs
LE65Gwx4BPHm9EqWHIBX6aPzbgbJZZ06f6jWVBi/N7e/5n8lkxXqS23DBKemapyu
S1i56sH7mQSMaRZP/iiOroAJemPNxv1IQkykxw2woWMmTLKLMCD/i+4DxejE50tK
dxaOLTc4HDCsattw/RVJO6fwE414IXHMv330z4HKWJevMQ+CmQGfswvCwgeBP9n8
PItLjBQAXIhJBBgRAgAJBQJSh85OAhsMAAoJEMbjGcM0QQaCzpAAmwUNoRyySf9p
5G3/2UD1PMueIwOtAKDVVDXEq5LJPVg4iafNu0SRMwgP0Q==
=icbY
-----END PGP PUBLIC KEY BLOCK-----
```

is this intended, or just a mistake?

**Comment by oxzi** [comment]
Thanks for creating this issue. Ubuntu Bionic Beaver, 18.04, is end of life since 31 May 2023. Thus, we are no longer building packages for it and this is an old artifact, which was not re-signed with the new key. Please excuse the inconveniences.

Personally, I would advise you to upgrade to a supported Ubuntu LTS release.

**Comment by C0rn3j** [comment]
Thanks, I was just wondering if this is intended!

**Comment by oxzi** [comment]
Yes, this is intended. However, maybe we should communicate more directly that packages are no longer being built when a distribution reaches its end of life and that the old packages are being kept there as they are to at least not break some aging setups out there.

Thus, thanks for coming forward as it is a good reminder to document this more prominently.

**Comment by C0rn3j** [comment]
I figured they are not being built, (they haven't been for a while on bionic, am aware it's EOL outside of ESM), I was just confused they are not being *re-signed*, which as far as I understand should be a possibility to do without a full package rebuild, but I may be misremembering how dpkg/deb works.

I would also not be sure where to obtain the key directly from Icinga.  
I suppose one can query the public key servers, I just copied it from a backup.
